README.md

```diff
@@ -1,20 +1,31 @@
-<p style="text-align:center;" align="center"><a href="https://layer5.io/meshery"><img align="center" style="margin-bottom:20px;" src="https://raw.githubusercontent.com/layer5io/meshery-kuma/master/img/readme/meshery-logo-light-text.svg"  width="70%" /></a><br /><br /></p>
+<p style="text-align:center;" align="center"><a href="https://layer5.io/meshery"><picture align="center">
+  <source media="(prefers-color-scheme: dark)" srcset="https://raw.githubusercontent.com/meshery/meshery/master/.github/assets/images/meshery/meshery-logo-dark-text-side.svg"  width="70%" align="center" style="margin-bottom:20px;">
```

Comment:
hey @pratul20 again this is good but let us replace this with relative referencing , which means store the image locally in `meshery-kuma` and then refer it.

Please look other repos for more details if needed.

Reply:
@legendarykamal I have added the logos to local directory of meshery kuma and provided their paths in README file. It will work fine after it gets merged into master. Because thats the path I have to give.